OCPBUGS-41156, SDN-4930, OCPBUGS-44794, OCPBUGS-43354, OCPBUGS-43519, OCPBUGS-32754: Downstream Merge [12-04-2024]

jluhrsen opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

All changes must adhere to this template to make it easy for reviewers
and preserve rationale/history behind every change
-->

## 📑 Description
<!-- Add a brief description of the pr -->

<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

## Additional Information for reviewers
<!--
What exactly did you change - you may also defer to information
contained in commit messages. At a bare minimum it's worth highlighting
which areas of the code were changed as it's easier to assign reviewers
-->

## ✅ Checks
<!-- Make sure your pr passes the CI checks and do check the following fields as needed - -->
- [ ] My code requires changes to the documentation
- [ ] if so, I have updated the documentation as required
- [ ] My code requires tests
- [ ] if so, I have added and/or updated the tests as required
- [ ] All the tests have passed in the CI <!-- If not leave a comment as to why the CI is red and if you need help understanding what's wrong -->

## How to verify it
<!--
Did you include unit tests? or end-to-end tests?
How can I manually verify that this patch achieves its objective
-->

jluhrsen commented:
/payload ci blocking
/payload nightly blocking

jluhrsen commented:
/payload 4.18 ci blocking
/payload 4.18 nighty blocking

jluhrsen commented:
/payload 4.18 ci blocking

jluhrsen commented:
/payload 4.18 nighty blocking

jluhrsen commented:
/payload 4.18 nighty blocking

jluhrsen commented:
/payload 4.18 nightly blocking

🤣

jluhrsen commented:
/retest

maiqueb commented:
/jira refresh

maiqueb commented:
/lgtm

maiqueb commented:
/retitle SDN-4930, OCPBUGS-44794: Downstream Merge [11-15-24]

maiqueb commented:
/jira refresh

jluhrsen commented:
/payload 4.18 ci blocking


/payload 4.18 nightly blocking

jluhrsen commented:
> @jluhrsen: trigger 4 job(s) of type blocking for the ci release of OCP 4.18
> 
> * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-aws-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-azure-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-e2e-gcp-ovn-upgrade
> * periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn
> 
> See details on https://pr-payload-tests.ci.openshift.org/runs/ci/d9cd1af0-a773-11ef-95df-d3b536c785d2-0
> 
> trigger 14 job(s) of type blocking for the nightly release of OCP 4.18
> 
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-upgrade-ovn-single-node
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-e2e-azure-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-gcp-ovn-rt-upgrade
> * periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn-conformance
> * periodic-ci-openshift-release-master-nightly-4.18-e2e-aws-ovn-serial
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-techpreview
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-techpreview-serial
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-upgrade
> * periodic-ci-openshift-release-master-nightly-4.18-fips-payload-scan
> * periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ipi-ovn-bm
> * periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ipi-ovn-ipv6
> * periodic-ci-openshift-microshift-release-4.18-periodics-e2e-aws-ovn-ocp-conformance
> * periodic-ci-openshift-microshift-release-4.18-periodics-e2e-aws-ovn-ocp-conformance-serial
> 
> See details on https://pr-payload-tests.ci.openshift.org/runs/ci/d9cd1af0-a773-11ef-95df-d3b536c785d2-1

These don't look good at all. looks like maybe some trouble getting cluster resources from the cloud provider.
trying again before spending time digging:

/payload 4.18 ci blocking

/payload 4.18 nightly blocking

jluhrsen commented:
/payload 4.18 ci blocking

/payload 4.18 nightly blocking

jluhrsen commented:
> > @jluhrsen: trigger 4 job(s) of type blocking for the ci release of OCP 4.18
> > 
> > * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-aws-ovn-upgrade
> > * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-azure-ovn-upgrade
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-gcp-ovn-upgrade
> > * periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn
> > 
> > See details on https://pr-payload-tests.ci.openshift.org/runs/ci/d9cd1af0-a773-11ef-95df-d3b536c785d2-0
> > trigger 14 job(s) of type blocking for the nightly release of OCP 4.18
> > 
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-upgrade-ovn-single-node
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-upgrade
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-azure-ovn-upgrade
> > * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-gcp-ovn-rt-upgrade
> > * periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn-conformance
> > * periodic-ci-openshift-release-master-nightly-4.18-e2e-aws-ovn-serial
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-techpreview
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-techpreview-serial
> > * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-upgrade
> > * periodic-ci-openshift-release-master-nightly-4.18-fips-payload-scan
> > * periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ipi-ovn-bm
> > * periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ipi-ovn-ipv6
> > * periodic-ci-openshift-microshift-release-4.18-periodics-e2e-aws-ovn-ocp-conformance
> > * periodic-ci-openshift-microshift-release-4.18-periodics-e2e-aws-ovn-ocp-conformance-serial
> > 
> > See details on https://pr-payload-tests.ci.openshift.org/runs/ci/d9cd1af0-a773-11ef-95df-d3b536c785d2-1
> 
> These don't look good at all. looks like maybe some trouble getting cluster resources from the cloud provider. trying again before spending time digging:
> 
> /payload 4.18 ci blocking
> 
> /payload 4.18 nightly blocking

I think whatever the infra issue was may be resolved just in the past 12h or so. found something similar
in [slack](https://redhat-internal.slack.com/archives/CBN38N3MW/p1732196862922219?thread_ts=1732159306.203639&cid=CBN38N3MW) which is supposed to be fixed now

/payload 4.18 ci blocking

/payload 4.18 nightly blocking

jluhrsen commented:
no idea what's going on. trying again:

/payload 4.18 ci blocking

/payload 4.18 nightly blocking


for example, [this aggregate job](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/aggregator-periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-upgrade/1859665143978266624) for 4.18-e2e-aws-ovn-upgrade has 10 jobs ([example](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/openshift-ovn-kubernetes-2357-ci-4.18-e2e-aws-ovn-upgrade/1859665114303565824)) that all failed to build and the error is something about not being able to install `openshift-clients`:
```
Error: 
 Problem: cannot install the best candidate for the job
  - package openshift-clients-4.18.0-202411201831.p0.g6490bf2.assembly.stream.el9.ppc64le from rhel-9-server-ose-ppc64le does not have a compatible architecture
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
error: build error: building at STEP "RUN INSTALL_PKGS=" 	openssl firewalld-filesystem 	libpcap iproute iproute-tc strace 	tcpdump iputils 	libreswan 	ethtool conntrack-tools 	openshift-clients 	" && 	dnf install -y --nodocs $INSTALL_PKGS && 	eval "dnf install -y --nodocs $(cat /more-pkgs)" && 	dnf clean all && rm -rf /var/cache/*": while running runtime: exit status 1

```

jluhrsen commented:
> @jluhrsen: trigger 4 job(s) of type blocking for the ci release of OCP 4.18
> 
> * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-aws-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-azure-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-e2e-gcp-ovn-upgrade
> * periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn
> 
> See details on https://pr-payload-tests.ci.openshift.org/runs/ci/d9b348e0-a909-11ef-9dd6-d521bb0d8055-0
> 
> trigger 13 job(s) of type blocking for the nightly release of OCP 4.18
> 
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-upgrade-ovn-single-node
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-e2e-azure-ovn-upgrade
> * periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-gcp-ovn-rt-upgrade
> * periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn-conformance
> * periodic-ci-openshift-release-master-nightly-4.18-e2e-aws-ovn-serial
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-techpreview
> * periodic-ci-openshift-release-master-ci-4.18-e2e-aws-ovn-techpreview-serial
> * periodic-ci-openshift-release-master-nightly-4.18-fips-payload-scan
> * periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ipi-ovn-bm
> * periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ipi-ovn-ipv6
> * periodic-ci-openshift-microshift-release-4.18-periodics-e2e-aws-ovn-ocp-conformance
> * periodic-ci-openshift-microshift-release-4.18-periodics-e2e-aws-ovn-ocp-conformance-serial
> 
> See details on https://pr-payload-tests.ci.openshift.org/runs/ci/d9b348e0-a909-11ef-9dd6-d521bb0d8055-1

for the most part it looks like the jobs that are failing from these payload requests
are running too long. even the azure-ovn-upgrade job we bumped to 5h30m is
timing out. I looked at aprox half of the jobs and a majority of them are passing
the e2e, but the job times out in some step after e2e. I think the actual builds are
taking a long time as well and might be a big reason we are hitting these time
limits. Will need to look closer, because this is not happening in the normal jobs
automatically triggered in the PR. not sure why the jobs triggered by the aggregate
job would be different.

tssurya commented:
I don't like this "context retried without replacement" message, are the jobs not being freshly triggered or what, -> I cannot even see the links of the passed jobs, I think they are too old now, like days old to make any worthy signals, I have to retry them all.

tssurya commented:
/test all

jluhrsen commented:
/retest

jluhrsen commented:
> I don't like this "context retried without replacement" message, are the jobs not being freshly triggered or what, -> I cannot even see the links of the passed jobs, I think they are too old now, like days old to make any worthy signals, I have to retry them all.

where did you see "context retried without replacement"? not following that.

jluhrsen commented:
/retest

tssurya commented:
> where did you see "context retried without replacement"? not following that.

I mean by the time you woke up I fixed it :P so you won't see it anymore

tssurya commented:
/retitle SDN-4930, OCPBUGS-44794, OCPBUGS-43354: Downstream Merge [11-15-24]

tssurya commented:
/jira refresh

maiqueb commented:
/jira refresh

jluhrsen commented:
/retest

jluhrsen commented:
/retest

pperiyasamy commented:
/retitle SDN-4930, OCPBUGS-44794, OCPBUGS-43354, OCPBUGS-43519: Downstream Merge [12-04-2024]

ricky-rav commented:
/retitle SDN-4930, OCPBUGS-44794, OCPBUGS-43354, OCPBUGS-43519, OCPBUGS-32754: Downstream Merge [12-04-2024]

ricky-rav commented:
/jira refresh

jluhrsen commented:
/retest

jluhrsen commented:
/retest
/payload 4.18 ci blocking
/payload 4.18 nighty blocking

jluhrsen commented:
wait, are we 4.19 now?

/payload 4.19 ci blocking
/payload 4.19 nighty blocking

jluhrsen commented:
/payload 4.18 ci blocking

jluhrsen commented:
/payload 4.18 nightly blocking

jluhrsen commented:
/retest

jluhrsen commented:
/test fips-payload-scan

jluhrsen commented:
/test e2e-metal-ipi-ovn-bm

jluhrsen commented:
/test e2e-aws-ovn-ocp-conformance

jluhrsen commented:
/test e2e-aws-ovn-ocp-conformance-serial

jluhrsen commented:
/payload-aggregate periodic-ci-openshift-release-master-ci-4.18-e2e-azure-ovn-upgrade 10

jluhrsen commented:
/payload 4.18 ci blocking


/payload 4.18 nightly blocking

jluhrsen commented:
/test e2e-aws-ovn-serial
/test e2e-aws-ovn-shared-to-local-gateway-mode-migration

jluhrsen commented:
This PR is totally broken now.

1) techpreview jobs go CrashLooping for ovnkube-node pods on install
2) I'm assuming the same problem is still there with the endpointslice retry failures causing Alerts to fire

jluhrsen commented:
> This PR is totally broken now.

/hold

> 1. techpreview jobs go CrashLooping for ovnkube-node pods on install

below is a snippet of the error I get from the ovnkube-controller in the ovnkube-node pod, but this line stood
out to me a little bit:

`ovnkube.go:137] failed to run ovnkube: [failed to start network controller: failed to start default network controller - while waiting for any node to have zone: "ip-10-0-77-83.us-west-1.compute.internal", error: context canceled, failed to start node network controller: failed to start default node network controller: failed to run kubelet restart tracker: failed to connect to systemd: dial unix /run/systemd/private: connect: no such file or directory]`

although the message you would get in 'oc describe' on the crashlooping pod does not contain that. I saw two
commits related to systemd:

1. [6d091b97a](https://github.com/openshift/ovn-kubernetes/pull/2357/commits/6d091b97aa0d792e9d174d1006c29b9a8d40489f) Mount systemd/private to the ovnkube containers.
2. [070f31109](https://github.com/openshift/ovn-kubernetes/pull/2357/commits/070f3110920f3f14f157005a0676f93af2722ded) vendor in systemd listener

1 seemed like it could be related, but I reverted it here on this PR to check and the problem persists. Just now
I reverted the "vendor" commit, but I don't know how that would matter.

Here is a full log of a crashlooping pod I got from clusterbot today:

```
❯ oc logs -nopenshift-ovn-kubernetes                           ovnkube-node-hjznl -c ovnkube-controller
+ . /ovnkube-lib/ovnkube-lib.sh
++ set -x
++ K8S_NODE=ip-10-0-77-83.us-west-1.compute.internal
++ [[ -n ip-10-0-77-83.us-west-1.compute.internal ]]
++ [[ -f /env/ip-10-0-77-83.us-west-1.compute.internal ]]
++ northd_pidfile=/var/run/ovn/ovn-northd.pid
++ controller_pidfile=/var/run/ovn/ovn-controller.pid
++ controller_logfile=/var/log/ovn/acl-audit-log.log
++ vswitch_dbsock=/var/run/openvswitch/db.sock
++ nbdb_pidfile=/var/run/ovn/ovnnb_db.pid
++ nbdb_sock=/var/run/ovn/ovnnb_db.sock
++ nbdb_ctl=/var/run/ovn/ovnnb_db.ctl
++ sbdb_pidfile=/var/run/ovn/ovnsb_db.pid
++ sbdb_sock=/var/run/ovn/ovnsb_db.sock
++ sbdb_ctl=/var/run/ovn/ovnsb_db.ctl
+ start-ovnkube-node 4 29103 29105
+ local log_level=4
+ local metrics_port=29103
+ local ovn_metrics_port=29105
+ [[ 3 -ne 3 ]]
+ cni-bin-copy
+ . /host/etc/os-release
++ NAME='Red Hat Enterprise Linux CoreOS'
++ ID=rhcos
++ ID_LIKE='rhel fedora'
++ VERSION=418.94.202412120450-0
++ VERSION_ID=4.18
++ VARIANT=CoreOS
++ VARIANT_ID=coreos
++ PLATFORM_ID=platform:el9
++ PRETTY_NAME='Red Hat Enterprise Linux CoreOS 418.94.202412120450-0'
++ ANSI_COLOR='0;31'
++ CPE_NAME=cpe:/o:redhat:enterprise_linux:9::baseos::coreos
++ HOME_URL=https://www.redhat.com/
++ DOCUMENTATION_URL=https://docs.okd.io/latest/welcome/index.html
++ BUG_REPORT_URL=https://access.redhat.com/labs/rhir/
++ REDHAT_BUGZILLA_PRODUCT='OpenShift Container Platform'
++ REDHAT_BUGZILLA_PRODUCT_VERSION=4.18
++ REDHAT_SUPPORT_PRODUCT='OpenShift Container Platform'
++ REDHAT_SUPPORT_PRODUCT_VERSION=4.18
++ OPENSHIFT_VERSION=4.18
++ RHEL_VERSION=9.4
++ OSTREE_VERSION=418.94.202412120450-0
+ rhelmajor=
+ case "${ID}" in
++ echo cpe:/o:redhat:enterprise_linux:9::baseos::coreos
++ cut -f 5 -d :
+ RHEL_VERSION=9
++ echo 9
++ sed -E 's/([0-9]+)\.{1}[0-9]+(\.[0-9]+)?/\1/'
+ rhelmajor=9
+ sourcedir=/usr/libexec/cni/
+ case "${rhelmajor}" in
+ sourcedir=/usr/libexec/cni/rhel9
+ cp -f /usr/libexec/cni/rhel9/ovn-k8s-cni-overlay /cni-bin-dir/
++ date '+%m%d %H:%M:%S.%N'
+ echo 'I1212 22:16:58.325034441 - disable conntrack on geneve port'
I1212 22:16:58.325034441 - disable conntrack on geneve port
+ iptables -t raw -A PREROUTING -p udp --dport 6081 -j NOTRACK
+ iptables -t raw -A OUTPUT -p udp --dport 6081 -j NOTRACK
+ ip6tables -t raw -A PREROUTING -p udp --dport 6081 -j NOTRACK
+ ip6tables -t raw -A OUTPUT -p udp --dport 6081 -j NOTRACK
++ date '+%m%d %H:%M:%S.%N'
I1212 22:16:58.332607240 - starting ovnkube-node
+ echo 'I1212 22:16:58.332607240 - starting ovnkube-node'
+ '[' shared == shared ']'
+ gateway_mode_flags='--gateway-mode shared --gateway-interface br-ex'
+ export_network_flows_flags=
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ gw_interface_flag=
+ '[' -d /sys/class/net/br-ex1 ']'
+ node_mgmt_port_netdev_flags=
+ [[ -n '' ]]
+ [[ -n '' ]]
+ multi_network_enabled_flag=
+ [[ true == \t\r\u\e ]]
+ multi_network_enabled_flag=--enable-multi-network
+ network_segmentation_enabled_flag=
+ [[ true == \t\r\u\e ]]
+ multi_network_enabled_flag=--enable-multi-network
+ network_segmentation_enabled_flag=--enable-network-segmentation
+ network_observability_enabled_flag=
+ [[ true == \t\r\u\e ]]
+ network_observability_enabled_flag=--enable-observability
+ multi_network_policy_enabled_flag=
+ [[ false == \t\r\u\e ]]
+ admin_network_policy_enabled_flag=
+ [[ true == \t\r\u\e ]]
+ admin_network_policy_enabled_flag=--enable-admin-network-policy
+ dns_name_resolver_enabled_flag=
+ [[ true == \t\r\u\e ]]
+ dns_name_resolver_enabled_flag=--enable-dns-name-resolver
+ ip_forwarding_flag=
+ '[' '' == Global ']'
+ ip_forwarding_flag=--disable-forwarding
+ sysctl -w net.ipv4.ip_forward=0
net.ipv4.ip_forward = 0
+ sysctl -w net.ipv6.conf.all.forwarding=0
net.ipv6.conf.all.forwarding = 0
+ NETWORK_NODE_IDENTITY_ENABLE=
+ [[ true == \t\r\u\e ]]
+ NETWORK_NODE_IDENTITY_ENABLE='
      --bootstrap-kubeconfig=/var/lib/kubelet/kubeconfig
      --cert-dir=/etc/ovn/ovnkube-node-certs
      --cert-duration=24h
    '
+ ovn_v4_join_subnet_opt=
+ [[ '' != '' ]]
+ ovn_v6_join_subnet_opt=
+ [[ '' != '' ]]
+ ovn_v4_masquerade_subnet_opt=
+ [[ 169.254.0.0/17 != '' ]]
+ ovn_v4_masquerade_subnet_opt='--gateway-v4-masquerade-subnet 169.254.0.0/17'
+ ovn_v6_masquerade_subnet_opt=
+ [[ fd69::/112 != '' ]]
+ ovn_v6_masquerade_subnet_opt='--gateway-v6-masquerade-subnet fd69::/112'
+ ovn_v4_transit_switch_subnet_opt=
+ [[ '' != '' ]]
+ ovn_v6_transit_switch_subnet_opt=
+ [[ '' != '' ]]
+ exec /usr/bin/ovnkube --init-ovnkube-controller ip-10-0-77-83.us-west-1.compute.internal --init-node ip-10-0-77-83.us-west-1.compute.internal --config-file=/run/ovnkube-config/ovnkube.conf --ovn-empty-lb-events --loglevel 4 --inactivity-probe=180000 --gateway-mode shared --gateway-interface br-ex --metrics-bind-address 127.0.0.1:29103 --ovn-metrics-bind-address 127.0.0.1:29105 --metrics-enable-pprof --metrics-enable-config-duration --export-ovs-metrics --disable-snat-multiple-gws --enable-multi-network --enable-network-segmentation --enable-admin-network-policy --enable-dns-name-resolver --enable-observability --enable-multicast --zone ip-10-0-77-83.us-west-1.compute.internal --enable-interconnect --acl-logging-rate-limit 20 --disable-forwarding --bootstrap-kubeconfig=/var/lib/kubelet/kubeconfig --cert-dir=/etc/ovn/ovnkube-node-certs --cert-duration=24h --gateway-v4-masquerade-subnet 169.254.0.0/17 --gateway-v6-masquerade-subnet fd69::/112
I1212 22:16:58.356156   16812 config.go:2268] Parsed config file /run/ovnkube-config/ovnkube.conf
I1212 22:16:58.356194   16812 config.go:2269] Parsed config: {Default:{MTU:8901 RoutableMTU:0 ConntrackZone:64000 HostMasqConntrackZone:0 OVNMasqConntrackZone:0 HostNodePortConntrackZone:0 ReassemblyConntrackZone:0 EncapType:geneve EncapIP: EncapPort:6081 InactivityProbe:100000 OpenFlowProbe:180 OfctrlWaitBeforeClear:0 MonitorAll:true OVSDBTxnTimeout:1m40s LFlowCacheEnable:true LFlowCacheLimit:0 LFlowCacheLimitKb:1048576 RawClusterSubnets:10.128.0.0/14/23 ClusterSubnets:[] EnableUDPAggregation:true Zone:global RawUDNAllowedDefaultServices:default/kubernetes,openshift-dns/dns-default UDNAllowedDefaultServices:[]} Logging:{File: CNIFile: LibovsdbFile:/var/log/ovnkube/libovsdb.log Level:4 LogFileMaxSize:100 LogFileMaxBackups:5 LogFileMaxAge:0 ACLLoggingRateLimit:20} Monitoring:{RawNetFlowTargets: RawSFlowTargets: RawIPFIXTargets: NetFlowTargets:[] SFlowTargets:[] IPFIXTargets:[]} IPFIX:{Sampling:400 CacheActiveTimeout:60 CacheMaxFlows:0} CNI:{ConfDir:/etc/cni/net.d Plugin:ovn-k8s-cni-overlay} OVNKubernetesFeature:{EnableAdminNetworkPolicy:true EnableEgressIP:true EgressIPReachabiltyTotalTimeout:1 EnableEgressFirewall:true EnableEgressQoS:true EnableEgressService:true EgressIPNodeHealthCheckPort:9107 EnableMultiNetwork:true EnableNetworkSegmentation:true DisableUDNHostIsolation:false EnableMultiNetworkPolicy:false EnableStatelessNetPol:false EnableInterconnect:false EnableMultiExternalGateway:true EnablePersistentIPs:false EnableDNSNameResolver:true EnableServiceTemplateSupport:false EnableObservability:false} Kubernetes:{BootstrapKubeconfig: CertDir: CertDuration:10m0s Kubeconfig: CACert: CAData:[] APIServer:https://api-int.ci-ln-cp5ln0k-76ef8.aws-2.ci.openshift.org:6443 Token: TokenFile: CompatServiceCIDR: RawServiceCIDRs:172.30.0.0/16 ServiceCIDRs:[] OVNConfigNamespace:openshift-ovn-kubernetes OVNEmptyLbEvents:false PodIP: RawNoHostSubnetNodes: NoHostSubnetNodes:<nil> HostNetworkNamespace:openshift-host-network DisableRequestedChassis:false PlatformType:AWS HealthzBindAddress:0.0.0.0:10256 CompatMetricsBindAddress: CompatOVNMetricsBindAddress: CompatMetricsEnablePprof:false DNSServiceNamespace:openshift-dns DNSServiceName:dns-default} Metrics:{BindAddress: OVNMetricsBindAddress: ExportOVSMetrics:false EnablePprof:false NodeServerPrivKey: NodeServerCert: EnableConfigDuration:false EnableScaleMetrics:false} OvnNorth:{Address: PrivKey: Cert: CACert: CertCommonName: Scheme: ElectionTimer:0 northbound:false exec:<nil>} OvnSouth:{Address: PrivKey: Cert: CACert: CertCommonName: Scheme: ElectionTimer:0 northbound:false exec:<nil>} Gateway:{Mode:shared Interface: EgressGWInterface: NextHop: VLANID:0 NodeportEnable:true DisableSNATMultipleGWs:false V4JoinSubnet:100.64.0.0/16 V6JoinSubnet:fd98::/64 V4MasqueradeSubnet:169.254.169.0/29 V6MasqueradeSubnet:fd69::/125 MasqueradeIPs:{V4OVNMasqueradeIP:169.254.169.1 V6OVNMasqueradeIP:fd69::1 V4HostMasqueradeIP:169.254.169.2 V6HostMasqueradeIP:fd69::2 V4HostETPLocalMasqueradeIP:169.254.169.3 V6HostETPLocalMasqueradeIP:fd69::3 V4DummyNextHopMasqueradeIP:169.254.169.4 V6DummyNextHopMasqueradeIP:fd69::4 V4OVNServiceHairpinMasqueradeIP:169.254.169.5 V6OVNServiceHairpinMasqueradeIP:fd69::5} DisablePacketMTUCheck:false RouterSubnet: SingleNode:false DisableForwarding:false AllowNoUplink:false} MasterHA:{ElectionLeaseDuration:137 ElectionRenewDeadline:107 ElectionRetryPeriod:26} ClusterMgrHA:{ElectionLeaseDuration:137 ElectionRenewDeadline:107 ElectionRetryPeriod:26} HybridOverlay:{Enabled:false RawClusterSubnets: ClusterSubnets:[] VXLANPort:4789} OvnKubeNode:{Mode:full DPResourceDeviceIdsMap:map[] MgmtPortNetdev: MgmtPortDPResourceName:} ClusterManager:{V4TransitSwitchSubnet:100.88.0.0/16 V6TransitSwitchSubnet:fd97::/64}}
I1212 22:16:58.357408   16812 certificate_store.go:130] Loading cert/key pair from "/etc/ovn/ovnkube-node-certs/ovnkube-client-current.pem".
I1212 22:16:58.357587   16812 certificate_store.go:130] Loading cert/key pair from "/etc/ovn/ovnkube-node-certs/ovnkube-client-current.pem".
I1212 22:16:58.357680   16812 certificate_manager.go:356] kubernetes.io/kube-apiserver-client: Certificate rotation is enabled
I1212 22:16:58.357698   16812 kube.go:390] Waiting for certificate
I1212 22:16:58.357720   16812 kube.go:397] Certificate found
I1212 22:16:58.357748   16812 certificate_manager.go:356] kubernetes.io/kube-apiserver-client: Certificate expiration is 2024-12-13 21:10:10 +0000 UTC, rotation deadline is 2024-12-13 15:08:34.123313241 +0000 UTC
I1212 22:16:58.357794   16812 certificate_manager.go:356] kubernetes.io/kube-apiserver-client: Waiting 16h51m35.7655223s for next certificate rotation
I1212 22:16:58.358655   16812 cert_rotation.go:140] Starting client certificate rotation controller
I1212 22:16:58.359314   16812 metrics.go:533] Starting metrics server at address "127.0.0.1:29103"
I1212 22:16:58.359690   16812 libovsdb.go:63] Client for Open_vSwitch using log verbosity 4 with lumberjack &lumberjack.Logger{Filename:"/var/log/ovnkube/libovsdb.log", MaxSize:100, MaxAge:0, MaxBackups:5, LocalTime:false, Compress:true, size:0, file:(*os.File)(nil), mu:sync.Mutex{state:0, sema:0x0}, millCh:(chan bool)(nil), startMill:sync.Once{done:atomic.Uint32{_:atomic.noCopy{}, v:0x0}, m:sync.Mutex{state:0, sema:0x0}}}
I1212 22:16:58.359762   16812 libovsdb.go:63] Client for OVN_Northbound using log verbosity 4 with lumberjack &lumberjack.Logger{Filename:"/var/log/ovnkube/libovsdb.log", MaxSize:100, MaxAge:0, MaxBackups:5, LocalTime:false, Compress:true, size:0, file:(*os.File)(nil), mu:sync.Mutex{state:0, sema:0x0}, millCh:(chan bool)(nil), startMill:sync.Once{done:atomic.Uint32{_:atomic.noCopy{}, v:0x0}, m:sync.Mutex{state:0, sema:0x0}}}
I1212 22:16:58.360082   16812 node_network_controller_manager.go:154] Starting the node network controller manager, Mode: full
I1212 22:16:58.360167   16812 factory.go:432] Starting watch factory
I1212 22:16:58.360283   16812 envvar.go:172] "Feature gate default state" feature="WatchListClient" enabled=false
I1212 22:16:58.360339   16812 envvar.go:172] "Feature gate default state" feature="InformerResourceVersion" enabled=false
I1212 22:16:58.360378   16812 reflector.go:305] Starting reflector *v1.Pod (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.360396   16812 reflector.go:341] Listing and watching *v1.Pod from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.360742   16812 reflector.go:305] Starting reflector *v1.NetworkPolicy (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.360786   16812 reflector.go:341] Listing and watching *v1.NetworkPolicy from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.360887   16812 reflector.go:305] Starting reflector *v1.Namespace (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.360927   16812 reflector.go:341] Listing and watching *v1.Namespace from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.361014   16812 reflector.go:305] Starting reflector *v1.Node (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.361052   16812 reflector.go:341] Listing and watching *v1.Node from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.361055   16812 reflector.go:305] Starting reflector *v1.EndpointSlice (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.361112   16812 reflector.go:341] Listing and watching *v1.EndpointSlice from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.361063   16812 reflector.go:305] Starting reflector *v1.Service (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.361193   16812 reflector.go:341] Listing and watching *v1.Service from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.367301   16812 libovsdb.go:63] Client for OVN_Southbound using log verbosity 4 with lumberjack &lumberjack.Logger{Filename:"/var/log/ovnkube/libovsdb.log", MaxSize:100, MaxAge:0, MaxBackups:5, LocalTime:false, Compress:true, size:0, file:(*os.File)(nil), mu:sync.Mutex{state:0, sema:0x0}, millCh:(chan bool)(nil), startMill:sync.Once{done:atomic.Uint32{_:atomic.noCopy{}, v:0x0}, m:sync.Mutex{state:0, sema:0x0}}}
I1212 22:16:58.367664   16812 metrics.go:533] Starting metrics server at address "127.0.0.1:29105"
I1212 22:16:58.373731   16812 network_controller_manager.go:308] Starting the ovnkube controller
I1212 22:16:58.373783   16812 network_controller_manager.go:313] Waiting up to 5m0s for NBDB zone to match: ip-10-0-77-83.us-west-1.compute.internal
I1212 22:16:58.373846   16812 network_controller_manager.go:333] NBDB zone sync took: 34.78µs
I1212 22:16:58.373878   16812 factory.go:432] Starting watch factory
I1212 22:16:58.375073   16812 reflector.go:368] Caches populated for *v1.NetworkPolicy from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.377670   16812 reflector.go:368] Caches populated for *v1.Namespace from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.377838   16812 reflector.go:368] Caches populated for *v1.EndpointSlice from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.378382   16812 reflector.go:368] Caches populated for *v1.Node from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.378560   16812 ovn_db.go:374] Found OVN DB Pod running on this node. Registering OVN DB Metrics
I1212 22:16:58.378636   16812 ovn_northd.go:123] Found OVNKube Master Pod running on this node. Registering OVN North Metrics
I1212 22:16:58.378383   16812 reflector.go:368] Caches populated for *v1.Service from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.381577   16812 ovn_db.go:329] /var/run/openvswitch/ovnnb_db.sock getting info failed: stat /var/run/openvswitch/ovnnb_db.sock: no such file or directory
I1212 22:16:58.381604   16812 ovn_db.go:326] ovnnb_db.sock found at /var/run/ovn/
I1212 22:16:58.386496   16812 reflector.go:368] Caches populated for *v1.Pod from k8s.io/client-go/informers/factory.go:160
I1212 22:16:58.394027   16812 ovn_db.go:421] Found db is standalone, don't register db_cluster metrics
I1212 22:16:58.461215   16812 reflector.go:305] Starting reflector *v1alpha1.AdminNetworkPolicy (0s) from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:58.461227   16812 reflector.go:341] Listing and watching *v1alpha1.AdminNetworkPolicy from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:58.461289   16812 reflector.go:305] Starting reflector *v1alpha1.BaselineAdminNetworkPolicy (0s) from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:58.461303   16812 reflector.go:341] Listing and watching *v1alpha1.BaselineAdminNetworkPolicy from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:58.463552   16812 reflector.go:368] Caches populated for *v1alpha1.AdminNetworkPolicy from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:58.463582   16812 reflector.go:368] Caches populated for *v1alpha1.BaselineAdminNetworkPolicy from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:58.474343   16812 reflector.go:305] Starting reflector *v1.EgressIP (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressip/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.474354   16812 reflector.go:341] Listing and watching *v1.EgressIP from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressip/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.476151   16812 reflector.go:368] Caches populated for *v1.EgressIP from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressip/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.561843   16812 reflector.go:305] Starting reflector *v1.EgressFirewall (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.561855   16812 reflector.go:341] Listing and watching *v1.EgressFirewall from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.564017   16812 reflector.go:368] Caches populated for *v1.EgressFirewall from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.574744   16812 reflector.go:305] Starting reflector *v1alpha1.DNSNameResolver (0s) from github.com/openshift/client-go/network/informers/externalversions/factory.go:116
I1212 22:16:58.574756   16812 reflector.go:341] Listing and watching *v1alpha1.DNSNameResolver from github.com/openshift/client-go/network/informers/externalversions/factory.go:116
I1212 22:16:58.576943   16812 reflector.go:368] Caches populated for *v1alpha1.DNSNameResolver from github.com/openshift/client-go/network/informers/externalversions/factory.go:116
I1212 22:16:58.662781   16812 reflector.go:305] Starting reflector *v1.EgressQoS (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressqos/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.662792   16812 reflector.go:341] Listing and watching *v1.EgressQoS from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressqos/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.664859   16812 reflector.go:368] Caches populated for *v1.EgressQoS from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressqos/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.675655   16812 reflector.go:305] Starting reflector *v1.EgressService (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressservice/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.675668   16812 reflector.go:341] Listing and watching *v1.EgressService from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressservice/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.677692   16812 reflector.go:368] Caches populated for *v1.EgressService from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressservice/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.763488   16812 reflector.go:305] Starting reflector *v1.AdminPolicyBasedExternalRoute (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/adminpolicybasedroute/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.763499   16812 reflector.go:341] Listing and watching *v1.AdminPolicyBasedExternalRoute from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/adminpolicybasedroute/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.765588   16812 reflector.go:368] Caches populated for *v1.AdminPolicyBasedExternalRoute from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/adminpolicybasedroute/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.776433   16812 reflector.go:305] Starting reflector *v1.NetworkAttachmentDefinition (0s) from github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/client/informers/externalversions/factory.go:117
I1212 22:16:58.776471   16812 reflector.go:341] Listing and watching *v1.NetworkAttachmentDefinition from github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/client/informers/externalversions/factory.go:117
I1212 22:16:58.778346   16812 reflector.go:368] Caches populated for *v1.NetworkAttachmentDefinition from github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/client/informers/externalversions/factory.go:117
I1212 22:16:58.863664   16812 reflector.go:305] Starting reflector *v1.UserDefinedNetwork (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.863677   16812 reflector.go:341] Listing and watching *v1.UserDefinedNetwork from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.863679   16812 reflector.go:305] Starting reflector *v1.ClusterUserDefinedNetwork (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.863688   16812 reflector.go:341] Listing and watching *v1.ClusterUserDefinedNetwork from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.865917   16812 reflector.go:368] Caches populated for *v1.ClusterUserDefinedNetwork from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.865922   16812 reflector.go:368] Caches populated for *v1.UserDefinedNetwork from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:58.876847   16812 network_controller_manager.go:344] Waiting up to 5m0s for a node to have "ip-10-0-77-83.us-west-1.compute.internal" zone
I1212 22:16:58.964220   16812 default_node_network_controller.go:161] Enable node proxy healthz server on 0.0.0.0:10256
I1212 22:16:58.964322   16812 default_node_network_controller.go:724] PreStarting the default node network controller
I1212 22:16:58.966801   16812 ovs.go:159] Exec(9): /usr/bin/ovs-vsctl --timeout=15 --no-heading --data=bare --format=csv --columns name list interface
I1212 22:16:58.969238   16812 ovs.go:162] Exec(8): stdout: ""
I1212 22:16:58.969252   16812 ovs.go:163] Exec(8): stderr: ""
I1212 22:16:58.969263   16812 node_network_controller_manager.go:344] CheckForStaleOVSInternalPorts took 5.139767ms
I1212 22:16:58.969272   16812 ovs.go:159] Exec(10): /usr/bin/ovs-vsctl --timeout=15 --columns=name,external_ids --data=bare --no-headings --format=csv find Interface external_ids:sandbox!="" external_ids:vf-netdev-name!=""
I1212 22:16:58.971142   16812 ovs.go:162] Exec(9): stdout: "br-int\nens5\nbr-ex\n"
I1212 22:16:58.971160   16812 ovs.go:163] Exec(9): stderr: ""
I1212 22:16:58.973457   16812 ovs.go:162] Exec(10): stdout: ""
I1212 22:16:58.973471   16812 ovs.go:163] Exec(10): stderr: ""
I1212 22:16:58.973899   16812 ovs.go:159] Exec(11): /usr/bin/ovn-sbctl --timeout=15 --no-leader-only get SB_Global . options:name
I1212 22:16:58.977242   16812 ovs.go:162] Exec(11): stdout: "ip-10-0-77-83.us-west-1.compute.internal\n"
I1212 22:16:58.977257   16812 ovs.go:163] Exec(11): stderr: ""
I1212 22:16:58.977282   16812 ovs.go:159] Exec(12): /usr/bin/ovs-vsctl --timeout=15 set Open_vSwitch . external_ids:ovn-encap-type=geneve external_ids:ovn-encap-ip=10.0.77.83 external_ids:ovn-remote-probe-interval=180000 external_ids:ovn-openflow-probe-interval=180 other_config:bundle-idle-timeout=180 external_ids:ovn-is-interconn=true external_ids:ovn-monitor-all=true external_ids:ovn-ofctrl-wait-before-clear=0 external_ids:ovn-enable-lflow-cache=true external_ids:ovn-set-local-ip="true" external_ids:ovn-memlimit-lflow-cache-kb=1048576 external_ids:hostname="ip-10-0-77-83.us-west-1.compute.internal"
I1212 22:16:58.981375   16812 ovs.go:162] Exec(12): stdout: ""
I1212 22:16:58.981389   16812 ovs.go:163] Exec(12): stderr: ""
I1212 22:16:58.981396   16812 ovs.go:159] Exec(13): /usr/bin/ovs-vsctl --timeout=15 -- clear bridge br-int netflow -- clear bridge br-int sflow -- clear bridge br-int ipfix
I1212 22:16:58.985214   16812 ovs.go:162] Exec(13): stdout: ""
I1212 22:16:58.985228   16812 ovs.go:163] Exec(13): stderr: ""
I1212 22:16:58.990412   16812 udn_isolation.go:103] Found kubelet cgroup path: system.slice/kubelet.service
E1212 22:16:59.022663   16812 node_network_controller_manager.go:171] Stopping node network controller manager, err=failed to start default node network controller: failed to run kubelet restart tracker: failed to connect to systemd: dial unix /run/systemd/private: connect: no such file or directory
I1212 22:16:59.022691   16812 network_attach_def_controller.go:161] [node-network-controller-manager NAD controller]: shutting down
I1212 22:16:59.022745   16812 metrics.go:553] Stopping metrics server at address "127.0.0.1:29105"
I1212 22:16:59.022772   16812 ovnkube.go:595] Stopping ovnkube...
I1212 22:16:59.022860   16812 reflector.go:311] Stopping reflector *v1.Service (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:59.022886   16812 reflector.go:311] Stopping reflector *v1.NetworkPolicy (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:59.022887   16812 reflector.go:311] Stopping reflector *v1.ClusterUserDefinedNetwork (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.022923   16812 reflector.go:311] Stopping reflector *v1.Pod (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:59.022941   16812 reflector.go:311] Stopping reflector *v1alpha1.DNSNameResolver (0s) from github.com/openshift/client-go/network/informers/externalversions/factory.go:116
I1212 22:16:59.022957   16812 factory.go:548] Stopping watch factory
I1212 22:16:59.022959   16812 reflector.go:311] Stopping reflector *v1.NetworkAttachmentDefinition (0s) from github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/client/informers/externalversions/factory.go:117
I1212 22:16:59.022994   16812 reflector.go:311] Stopping reflector *v1.AdminPolicyBasedExternalRoute (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/adminpolicybasedroute/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.023009   16812 reflector.go:311] Stopping reflector *v1alpha1.BaselineAdminNetworkPolicy (0s) from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:59.023064   16812 reflector.go:311] Stopping reflector *v1.EgressService (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressservice/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.023099   16812 reflector.go:311] Stopping reflector *v1.EndpointSlice (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:59.023108   16812 reflector.go:311] Stopping reflector *v1.EgressQoS (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressqos/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.023138   16812 reflector.go:311] Stopping reflector *v1.EgressIP (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressip/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.023179   16812 reflector.go:311] Stopping reflector *v1alpha1.AdminNetworkPolicy (0s) from sigs.k8s.io/network-policy-api/pkg/client/informers/externalversions/factory.go:141
I1212 22:16:59.023239   16812 reflector.go:311] Stopping reflector *v1.UserDefinedNetwork (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/userdefinednetwork/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.023267   16812 reflector.go:311] Stopping reflector *v1.Node (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:59.023304   16812 reflector.go:311] Stopping reflector *v1.Namespace (0s) from k8s.io/client-go/informers/factory.go:160
I1212 22:16:59.023323   16812 reflector.go:311] Stopping reflector *v1.EgressFirewall (0s) from github.com/openshift/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/informers/externalversions/factory.go:140
I1212 22:16:59.023339   16812 ovnkube.go:599] Stopped ovnkube
I1212 22:16:59.023353   16812 metrics.go:553] Stopping metrics server at address "127.0.0.1:29103"
F1212 22:16:59.023389   16812 ovnkube.go:137] failed to run ovnkube: [failed to start network controller: failed to start default network controller - while waiting for any node to have zone: "ip-10-0-77-83.us-west-1.compute.internal", error: context canceled, failed to start node network controller: failed to start default node network controller: failed to run kubelet restart tracker: failed to connect to systemd: dial unix /run/systemd/private: connect: no such file or directory]
```


> 2. I'm assuming the same problem is still there with the endpointslice retry failures causing Alerts to fire

bug to get this figured out is [here in this u/s issue](https://github.com/ovn-kubernetes/ovn-kubernetes/issues/4896)

trozet commented:
/jira refresh

jluhrsen commented:
/test e2e-aws-ovn-local-gateway
/test e2e-aws-ovn-serial
/test e2e-gcp-ovn-techpreview
/test e2e-metal-ipi-ovn-dualstack
/test e2e-metal-ipi-ovn-ipv6

trozet commented:
Results look good. Let's run payload

/payload 4.19 ci blocking
/payload 4.19 nightly blocking

trozet commented:
results from payload jobs look good

/lgtm
/hold cancel

trozet reviewed (approved):
/approve

maiqueb commented:
/retitle OCPBUGS-45115, OCPBUGS-41156, SDN-4930, OCPBUGS-44794, OCPBUGS-43354, OCPBUGS-43519, OCPBUGS-32754: Downstream Merge [12-04-2024]

ricky-rav commented:
/retitle OCPBUGS-55014,OCPBUGS-37193, OCPBUGS-45115, OCPBUGS-41156, SDN-4930, OCPBUGS-44794, OCPBUGS-43354, OCPBUGS-43519, OCPBUGS-32754: Downstream Merge [12-04-2024]
